### DISCUSSION TITLE
Using own robot model

### DISCUSSION THREAD

**Discussion by marcfuncke** [FieldRobotEvent/Virtual_Field_Robot_Event#8] [General]
Hey there,

is it allowed to use a model of our robot in the online event ? We would like to use different sensors (depth cameras) as we would use in the real event with our own robot. 

Best regards!

**Comment by ThijsRuigrok** [comment]
Yes, in task 3 and 4, it is allowed to use custom robot models, including sensors and actuators.

**Reply by marcfuncke** [reply]
Thanks for your answer!
But what about task 1 and 2? Is it mandatory that we use the given model?

**Reply by ThijsRuigrok** [reply]
Sorry for the late reply. For task 1 and 2 it is indeed mandatory to use the given model.
